### PR TITLE
Add tooltip to homepage sub-header

### PR DESCRIPTION
**Pull request by faihegberg**
This PR adds a tooltip to the homepage sub-header stating "UPDATED ON [DATE]", which describes when data is typically updated. The code also uses localized time based on the user's browser.

### CODE REVIEW

**Comment by pnavarrc** [comment]
The info icon is a bit off-center, maybe we should make the `span` with the date a `div` with display flex 

image

File: src/components/Header/HomePageHeader.tsx [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
+import { trackOpenTooltip } from 'components/InfoTooltip';
+
+function renderInfoTooltip(timeString: string): ReactElement {
+  // TODO (Fai): Migrate this content to CMS when possible.
+  // const { body } = homePageHeaderTooltipContent;
+  const body =
+    'We aim to update our data by ' +
+    timeString +
+    ' daily. Occasionally, when additional review is required, an update can be delayed by several hours. Note that certain data sources that we use (eg. ICU hospitalizations) are only updated once per week.';
+  return (
```
Comment (pnavarrc) [inline src/components/Header/HomePageHeader.tsx:17]: Using string interpolation is a bit more readable

```ts
const body = `We aim to update our data by ${timeString} daily. ...`;
```

**Review by pnavarrc — COMMENTED** [review]
Some notes

**Review by pnavarrc — APPROVED** [review]
Looks good!

**Comment by faihegberg** [comment]
[Tooltip annotate homepage "Update on X"](https://trello.com/c/IzoM5cMH/991-tooltip-annotate-homepage-update-on-x)

**Pull request merged**